src/playground/App.js

```diff
@@ -63,6 +64,9 @@ const App = () => {
 
     const lint = () => {
         try {
+            const validator = new Legacy.ConfigValidator();
```

Comment:
This doesn't show errors for invalid options, e.g., `quotes: ["error", "foo"]`.

The reason is that `@eslint/eslintrc` does not have access to rule definitions, so `ConfigValidator` will skip validating rule options unless we provide [`builtInRules`](https://github.com/eslint/eslintrc/blob/a57d60877c12601a7f012710a0be2bda8088a947/lib/shared/config-validator.js#L41) to the constructor.

```suggestion
            const validator = new Legacy.ConfigValidator({ builtInRules: rules });
```

Reply:
Fixed.